deno.json

```diff
@@ -13,5 +16,6 @@
     "test": "deno test -A --unstable tests/ src/",
     "build-action": "deno bundle ./src/utils/mod.ts > ./action/deps.js",
     "version-match": "deno run --allow-read --allow-env ./tools/version_match.ts"
-  }
+  },
+  "importMap": "./import_map.json"
```

Comment:
Out of curiosity - why import map and not `imports`?

Reply:
that's what `deno vendor` put there

Reply:
Of course 🤦